src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -225,6 +225,12 @@ def get_color(
             return "info"
     if event_for_tags:
         color: str | None = event_for_tags.get_tag("level")
+        if (
+            hasattr(event_for_tags, "occurrence")
+            and event_for_tags.occurrence is not None
+            and hasattr(event_for_tags.occurrence, "level")
```

Comment:
Hasattr will always be true here, since the property will always exist regardless of whether it's `None`

Reply:
Actually was referring to `level` here rather than the occurrence, although I guess it's true for both.

Reply:
oops yeah, updated

Reply:
Agh actually if anything falls into that block right now while we don't have a guarantee that the occurrence will be on the event, it'll raise an `AttributeError`.

Reply:
Right, I think the `hasattr(... occurrence)` check was ok to avoid that. Although I don't know why we'd have an `Event` arriving there anyway, other than in tests